src/components/dashboard/charts/cardLineChart.tsx

```diff
@@ -2,137 +2,67 @@ import React from 'react';
-                            borderDashOffset: [2],
-                            color: 'rgba(33, 37, 41, 0.3)',
-                            zeroLineColor: 'rgba(0, 0, 0, 0)',
-                            zeroLineBorderDash: [2],
-                            zeroLineBorderDashOffset: [2]
-                        }
-                    },
+interface ChartLineProps {
+    data: ChartLineData[];
+}
 
-                    y: {
-                        ticks: {
-                            fontColor: 'rgba(255,255,255,.7)'
-                        },
-                        display: true,
-                        title: {
-                            display: false,
-                            text: 'Value',
-                            fontColor: 'white'
-                        },
-                        grid: {
-                            borderDash: [3],
-                            borderDashOffset: [3],
-                            drawBorder: false,
-                            color: 'rgba(255, 255, 255, 0.15)',
-                            zeroLineColor: 'rgba(33, 37, 41, 0)',
-                            zeroLineBorderDash: [2],
-                            zeroLineBorderDashOffset: [2]
-                        }
-                    }
-                }
-            }
-        } as unknown) as ChartConfiguration; //todo: change this
+const CardLineChart = ({ data }: ChartLineProps) => {
+    const datasets = data.map((d) => {
+        return {
+            label: d.username,
+            backgroundColor: d.color,
+            borderColor: d.color,
+            data: d.data,
+            fill: false
+        };
+    });
 
+    React.useEffect(() => {
         const canvas = document.getElementById('line-chart') as HTMLCanvasElement;
         let ctx = canvas.getContext('2d')!;
-        window.myLine = new Chart(ctx, config);
+        window.myLine = new Chart(ctx, getConfig(datasets));
     }, []);
     return (
-        <>
-            <div className="bg-gray-800 relative flex flex-col min-w-0 break-words w-full shadow-lg rounded">
-                <div className="rounded-t mb-0 px-4 py-3 bg-transparent">
-                    <div className="flex flex-wrap items-center">
-                        <div className="relative w-full max-w-full flex-grow flex-1">
-                            <h6 className="uppercase text-white mb-1 text-xs font-semibold">Overview</h6>
-                            <h2 className="text-white text-xl font-semibold">Sales value</h2>
-                        </div>
+        <div className="bg-gray-800 relative flex flex-col min-w-0 break-words w-full shadow-lg rounded-lg">
+            <div className="rounded-t mb-0 px-4 py-3 bg-transparent">
+                <div className="flex flex-wrap items-center">
+                    <div className="relative w-full max-w-full flex-grow flex-1">
+                        <h2 className="text-white text-xl font-semibold text-center">Postępy w czasie</h2>
```

Comment:
I think like most of the styles are not necessary for chart header.

Reply:
Chart title should refer to it's role

Reply:
So, what title do you suggest?

Reply:
IDK. Scoreboard? Points chart? User points? Be creative ;)